src/modules/colorPicker/ColorPickerUI/Behaviors/AppearAnimationBehavior.cs

```diff
@@ -19,7 +19,6 @@ protected override void OnAttached()
 
         private void AssociatedObject_Loaded(object sender, RoutedEventArgs e)
         {
-            Appear();
```

Comment:
Nice one. This one unneeded because it works on IsVisibleChanged, right? Nice spotting.

Reply:
Yes